### ISSUE TITLE
[BUG] error when synthesis

### ISSUE THREAD

**Issue by hhhsiang** [openhwgroup/cva6#2544]
### Is there an existing CVA6 bug for this?

- [X] I have searched the existing bug issues

### Bug Description

When I synthesize cv64a6 core with `cv64a6_imafdc_sv39_config_pkg.sv` with Design Compiler version-2023.12, there are lots of error occurred, and all of them like the following：
![image](https://github.com/user-attachments/assets/4ba7ccbe-7e3a-44cd-93ff-1161d4b68f58)

> Error: ../..//core/scoreboard.sv:303: Constant expression too complex. (ELAB-937)

It's all about "Constant expression too complex. (ELAB-937)". I check the systemverilog code where the error happened. All of the errors seem to be caused by the parameter of cva6 config pkg.

![image](https://github.com/user-attachments/assets/2f7ff4db-c5f0-472c-831d-54531ba3c129)

When the parameters of cva6_config_pkg are used to determine whether a circuit or a module will be instantiated, the "Constant expression too complex. (ELAB-937)" error occurred. Lots of modules are not synthesized. How can I fixed it?

Any advice or suggestions would be greatly appreciated. Thanks in advance!

**Comment by JeanRochCoulon** [comment]
Hello @hhhsiang     The synthesis is run on cv32a65x, no experience on 64 bit configuration.
Do you experience the same type of error on cv32a65x configuration ?

**Comment by hhhsiang** [comment]
Hi @JeanRochCoulon , as far as i know, the cv64a6 was synthesized in the past, so there are scripts in Makefile under the `cva6/pd/synth` directory. Because in the old commit cv64a6 is the synthesis target in the makefile, I'm confused why did you say no experience on 64 bit configuration?

I also tyr to synthesize cv32a65x, with `make -C pd/synth cva6_synth FOUNDRY_PATH=/your/techno/basepath/ TECH_NAME=yourTechnoName TARGET_LIBRARY_FILES="yourLib1.db\ yourLib2.db" PERIOD=10 NAND2_AREA=650 TARGET=cv64a6_imafdc_sv39 ADDITIONAL_SEARCH_PATH="others/libs/paths/one\ others/libs/paths/two"`, but there is error occurred like the following：
![image](https://github.com/user-attachments/assets/4b9c8606-ac2f-47f2-a33d-d1e87fa40a2a)
 It seems that `$(SYNTH_FLOW_NAME)` parameter is not set properly.

**Comment by JeanRochCoulon** [comment]
Yes, you are right, I personally setup maybe 3 years ago the scripts to synthetize 32 and 64 bits configurations. But we currently use different scripts to do it. That is why these scripts are no more used.

**Comment by hhhsiang** [comment]
Understood. Could you provide the script you use for synthesizing the 64-bit configuration? I think it is difficult to construct a script to successfully synthesize cv64a6 core because there are so many dependeUnderstood. Could you provide the script you use for synthesizing the 64-bit configuration? I think it is difficult to construct a script to successfully synthesize cv64a6 core because there are so many dependent file need to be include and preprocess.nt file need to be include and preprocess.

**Comment by JeanRochCoulon** [comment]
Unfortunately, physical implementation depends on techno which we cannot disclose it.

**Comment by JeanRochCoulon** [comment]
@hhhsiang We are cleaning the "Github Issues". I assume this issue is closed. May I ask you to close it ?

**Comment by hhhsiang** [comment]
@ergulfaruk , you need to make sure that the parameters which is used to instantiate modules has no "dot", such as "CVA6Cfg.FpPresent" or "CVA6Cfg.NrCommitPorts" in my exmaples. You need to change them into constant.

**Comment by cainria** [comment]
Can we close this issue?
